superset-frontend/packages/generator-superset/generators/plugin-chart/templates/babel.config.erb

```diff
@@ -0,0 +1,20 @@
+const { getConfig } = require('@airbnb/config-babel');
```

Comment:
Is it possible to use babel config without `airbnb/config-babel`?

Reply:
It seems to bring in a bunch of good defaults (would otherwise have to do lots of other configs here), so I think it's a good idea to use it.